001-intro.md

```diff
@@ -100,96 +100,358 @@ to real life cases involving actual DMs.**
+  data models, as long as such Data Models are expressed according to the
+  VO-DML standard [@vodml]. In other terms, they can *serialize* data model
+  *instances* in a standard, interoperable way. Some examples are provided below
+  as concrete use cases.
+  * Service clients to faithfully reconstruct the semantics of the data model instances
+  they consume in VOTable format. Some concrete examples are also provided below.
+
+One of the main goals of this specification is also to alleviate data modelers
+from the burden of defining special serialization strategies for their data models, at least
+in most cases. Specialized serializations might be defined if there are special
+constraints in term of efficiency or effectiveness which require
+specialized serialization schemes.
+
+As a corollary to the above paragraph, client applications can also be implemented
+on top of standardized Input/Output libraries that implement the present specification,
+as the serialization mechanisms are standardized across data models. Without this specification
+clients would need to be coded against specialized serializations for each data model.
+Similarly, data providers can now serialize instances of any data model to VOTable using
+the same annotation mechanism.
+
+As a result, this mapping specification should enable a large number of concrete use cases
+to be implemented, by reducing the annotation burden on both data providers and data consumers,
+improving the overall interoperability, at least for what VOTable is concerned.
+
+This document also represents a template for mapping data model instances to other
+formats (see [@sec:other-formats]).
+
+Concrete Use Cases
+------------------
+
+### STC clients ###
+A typical usage scenario may be a VOTable client
+that is sensitive to certain models only, say Space Time Coordinates (STC).
+Such a tool may be
+written to understand annotations for instance of STC types, manipulate such instances, and write them back
+to disk.
+
+By finding an element
+mapped to a type definition from the STC model, the application may infer for example that it
 represents a coordinate on the sky and use this information according to
-its requirements.
+its requirements. For example, the client may convert all positions
+expressed in a certain coordinate frame to a different coordinate frame
+through some specific transform.
 
-Such a tool would not necessarily understand other models where such an
-STC type is *used* as a role. So, if the annotation refers to both the
-attribute’s role *and* type, even a naïve client can trivially find the
-information it needs. A more advanced client may want to read the Data
-Model Description File that describes the Data Model in a standardized,
-machine readable, fashion.
+Note that the client may never parse any VO-DML description files,
+as the knowledge about the data model may be assumed when the client code is developed.

```

Comment:
@lmichel 
Your comment: 
> Paragraph #3 in section 2.2.1. We can append: "...or simply by interpreting UCDs"

However, UCDs are not part of this specification and are never used as part of the mapping.